docs/content/preview/yugabyte-voyager/release-notes.md

```diff
@@ -13,6 +13,42 @@ type: docs
 
 What follows are the release notes for the YugabyteDB Voyager v1 release series. Content will be added as new notable features and changes are available in the patch releases of the YugabyteDB v1 series.
 
+## v1.8.10 - January 28, 2025
+
+### Enhancements
+
+- **Air-gapped Installation:** The air-gapped installation method has been improved to no longer require `gcc` on the client machine running `yb-voyager`.
+- **Enhanced Assessment and Schema Analysis reports:**
+  - The **Assessment Report** has been enhanced with a single section summarizing all issues in a table format. Each issue includes a summary and an expandable section for more details. Users can now sort issues based on criteria such as category, type, or impact.
+  - Detects the following unsupported features from PostgreSQL -
+    - SQL Body in Create Function
+    - Common Table Expressions (With queries) with MATERIAZED clause.
```

Comment:
With queries? What does this mean and why is With capitalized? @priyanshi-yb

Reply:
With queries is a command [syntax](https://www.postgresql.org/docs/current/queries-with.html#QUERIES-WITH) in PG. Maybe we can change it to full caps `WITH` to represent it as a Keyword in PG.

Reply:
Oh that WITH - yes, all caps